packages/sources/svelte/src/components/navigation/VtmnNavbar/VtmnNavbar.svelte

```diff
@@ -39,9 +32,9 @@
   {/if}
 
   {#if checkSlotExists('logo')}
-    <a class="vtmn-navbar_logo" href={logoHref}>
```

Comment:
Also change it on : 
- packages/showcases/css/stories/components/navigation/navbar/examples/overview.html
- packages/sources/vue/src/components/navigation/VtmnNavbar/VtmnNavbar.vue

React is already a `<div>`

Reply:
Done